NEWS.md

```diff
@@ -19,6 +19,8 @@
 
```

Comment:
(Request unrelated to this line of code. Just handy to have a `Resolve conversation` button when the task is complete.)

------------------------

Please call `devtools::document()` to update the `DESCRIPTION`'s `Collate` field. 
This will fix the `routine/routine` GitHub Actions check.

Reply:
(I think I did everything... I also looked at devtools::document - but that just had changes for `CairoPNG` - not related to this PR)